sdk/attestation/Azure.Security.Attestation/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Release History
+
 ## 1.0.0-beta.1 (Unreleased)
+Released as beta, not alpha.
```

Comment:
I'm not sure that I understand what this is describing.   Do you mean to say that this release will contain the same feature set as the alpha but is just being advanced to beta status?

Reply:
Apparently Azure releases are never alphas, they're only betas.